Physics/Collisions/CharacterControllers.cs

```diff
@@ -865,8 +865,8 @@ public void EnsureCapacity(int characterCapacity, int bodyHandleCapacity)
     /// <param name="bodyHandleCapacity">Target number of body handles to allocate space for.</param>
     public void Resize(int characterCapacity, int bodyHandleCapacity)
     {
-        int lastOccupiedIndex = -1;
```

Comment:
why int -> var

Reply:
Thut's not me, VS went nuts. Converting PR to draft until this is resolved